types/fluent-ffmpeg/index.d.ts

```diff
@@ -279,7 +287,7 @@ declare namespace Ffmpeg {
         /* tslint:disable:unified-signatures */
         ffprobe(callback: (err: any, data: FfprobeData) => void): (err: any, data: FfprobeData) => void;
         ffprobe(index: number, callback: (err: any, data: FfprobeData) => void): (err: any, data: FfprobeData) => void;
-        ffprobe(options: string[], callback: (err: any, data: FfprobeData) => void): (err: any, data: FfprobeData) => void;
```

Comment:
looks like string is supported.. see https://github.com/fluent-ffmpeg/node-fluent-ffmpeg/blob/b6b6ef35d3c09d6dc71a23ceb1fc25bccb7acb1f/lib/ffprobe.js#L119